cmd2/cmd2.py

```diff
@@ -1211,124 +1211,55 @@ def visible_prompt(self) -> str:
         return ansi.strip_style(self.prompt)
 
     def print_to(
```

Comment:
This is a lot cleaner here. Now `print_to` is doing one thing and doing it well.  I'd recommend adding a brief docstring to explain the purpose.

Reply:
Added docstring.